docs/mp/restclient/09_rest-client.adoc

```diff
@@ -20,10 +20,27 @@
 :h1Prefix: MP
 :description: Helidon MP Rest Client
 :feature-name: MicroProfile Rest Client
```

Comment:
Is the official name of this feature the Helidon MicroProfile Rest Client or do we refer to the generic 'rest client' after introducing it? Is it a proper Helidon feature or is it part of the MP spec that we use?
Rest Client vs. rest client throughout the doc

Reply:
Official spec name is `MicroProfile Rest Client`, but since that is pretty long to say it every time when you refer to it, I think that sticking with just Rest Client or rest client is fine. Helidon there just means that it is another of our libraries. Do you think Rest Client or rest client would be more proper to use?

Reply:
I have replaced occurrences of `rest client interface` to simply `client interface`. This can be revisited as part of https://github.com/oracle/helidon/issues/4462